src/assets/_project/_blocks/components/forms/_qg-forms-io.scss

```diff
@@ -2,7 +2,7 @@
 styles for forms.io forms
  */
 
-#qg-primary-content {
+#qg-primary-content form , #qg-primary-content #formio {
```

Comment:
is #qg-primary-content form really needed as that will affect all other forms not just form.io.

Reply:
> since the first merge broke other forms, please ensure this only targets form.io,

Currently the plan is to have it for all the forms 
@manjeetkgov  and @gvanimurthy

Reply:
then it should not be in the _qg-forms-io.scss, it should be in the _go-forms.scss as it targets all.

Reply:
I have removed _qg-forms.io and added all in the _qg-forms file

Reply:
I have changed the target to 'formsio' for now, will extend this further to cover SWE forms in the latest release.